README.md

```diff
@@ -841,6 +841,61 @@ Going crazy:
 ; => {:x 24}
 ```
 
+## To and from JSON
+
+The `m/encode` and `m/decode` functions work on clojure data. To go
+from clojure data to JSON, you need a JSON library like
+[jsonista](https://github.com/metosin/jsonista). Additionally, since
+`m/decode` doesn't check the schema, you need to run `m/validate` (or
+`m/explain`) if you want to make sure your data conforms to your
+schema.
```

Comment:
there is a fast `coercer` function described in https://www.metosin.fi/blog/high-performance-schemas-in-clojurescript-with-malli-1-2/#transformation, would it make sense to built that? pure transformers created ahead-of-time. up to you.

Reply:
I'll add something about that.